packages/wrangler/src/autoconfig/frameworks/tanstack.ts

```diff
@@ -0,0 +1,143 @@
+import assert from "assert";
+import path from "path";
```

Comment:
```suggestion
import assert from "node:assert";
import path from "node:path";
```

I'm surprised we don't have a lint rule for this  😮

Reply:
https://github.com/cloudflare/workers-sdk/pull/11411 🙂